### ISSUE TITLE
Add audio visualizer?

### ISSUE THREAD

**Issue by robegamesios** [jnthas/clockwise#50]
Hi is it possible to add an audio visualizer? Like using an inmp441 for sound input? Thanks

**Comment by jnthas** [comment]
Yes, it is! We can create a new clockface that uses this sensor. Do you have example that we can use as a start point?
